pennylane/tape/operation.py

```diff
@@ -96,6 +96,9 @@ def tensor_init(self, *args):
 
         try:
             QueuingContext.update_info(o, owner=self)
+        except ValueError:
+            o.queue()
```

Comment:
Is there any (present or future) situation where the operator is not supposed to be in the queue, and what should happen is that the annotation is not attempted?

Reply:
And would it be safer to match the message of the  ValueError?

Reply:
> Is there any (present or future) situation where the operator is not supposed to be in the queue, and what should happen is that the annotation is not attempted?

Good question. I don't think that should ever be the case, e.g.,

```python
op = qml.PauliX(0) @ qml.PauliY(1)

with QueuingContext():
    qml.expval(op)
```

To me, that would be intuitive if it just didn't do anything since `op` was not previously queued.